client/tests/integration/smartcontracts/validator_with_admin/src/lib.rs

```diff
@@ -4,18 +4,47 @@
 #![no_std]
 #![no_main]
 
+use iroha_validator::{parse, prelude::*};
+
 #[cfg(not(test))]
 extern crate panic_halt;
 
-use iroha_validator::{parse, prelude::*};
+/// Default validator
+///
+/// You can put state in your validator and use that state
+struct Validator;
 
-/// Allow operation if authority is `admin@admin` and fallback to
-/// [`iroha_default_validator::validate()`] if not.
+impl DefaultValidator for Validator {}
+
+/// Allow operation if authority is `admin@admin` and if not,
+/// fallback to [`iroha_default_validator::validate()`].
 #[entrypoint(params = "[authority, operation]")]
-pub fn validate(
-    authority: <Account as Identifiable>::Id,
-    operation: NeedsValidationBox,
-) -> Verdict {
+pub fn validate(authority: AccountId, operation: NeedsValidationBox) -> Verdict {
```

Comment:
I changed my mind. I can restore it, is it relevant?

Reply:
Actually i like `AccountId` more than `<Account as Identifiable>::Id`.

Reply:
@Erigara , me too, I just don't understand why we've pushed another option so hard before

Reply:
I wonder why it's not relevant anymore

Reply:
> I just don't understand why we've pushed another option so hard before

because I'm trying to enforce some code style here and there and sometimes I make a dubious call 

it's just a matter of style. I wanted to use `Account::Id` but it always requires full path

Reply:
Alright, let's go with `AccountId`